lib/cli/utils.ts

```diff
@@ -4,6 +4,7 @@ import os from 'os';
 import path from 'path';
 
 const SATOSHIS_PER_COIN = 10 ** 8;
+const MAX_PRECISION = 12;
```

Comment:
Maybe call this `MAX_PRICE_DECIMAL_PLACES`. "Decimal place" is more accurate in this context I believe because "precision" can refer to the number of significant digits either to the left or right of the decimal point.